### ISSUE TITLE
Chore: Move cmd to root directory

### ISSUE THREAD

**Issue by marcusolsson** [grafana/grafana#19816]
**What would you like to be added**:

The `pkg` is normally reserved for libraries as opposed to applications. The `cmd` should be moved from `pkg` to the root directory.

[Project Layout](https://github.com/golang-standards/project-layout)

**Why is this needed**:

We want to make it as easy as possible to navigate the repository. Having `cmd` in `pkg` goes against common practices in the Go community. Go developers will look for a `cmd` directory in the root directory. If not, they will have to look for it.

**Comment by aknuds1** [comment]
Just adding the note that from my research, there's no standard in Go projects with regards to the _pkg_ directory. Many use it for public libraries, but it's _not_ a standard convention. As such, there's nothing wrong with how we currently structure the Grafana Go code.

**Comment by marcusolsson** [comment]
When I wrote this, I was referring to how pkg is commonly used together with cmd and internal to organize packages for Go applications, such as in [kubernetes](https://github.com/kubernetes/kubernetes) and [moby](https://github.com/moby/moby) (Docker).

Since then I've learned that Grafana and related projects—being multi-language projects—uses the pkg directory mainly to separate the Go code from Typescript code, making it the root directory for the parts of the project that are written in Go. With that, I agree that it's not worth moving cmd to the root directory.

While I think that using the same name as a common pattern invites unnecessary confusion, I don't think it's worth risking breaking things that rely on this. I'm closing this issue.
